CODEOWNERS

```diff
@@ -1,4 +1,3 @@
 /blade/ @chaitanyadeorukhkar @kamleshchandnani @dhruvdutt @anuraghazra @nashcheez @divyanshu013 @saurabhdaware
```

Comment:
only keep this? can remove everything else now

Reply:
Yep removed. not sure why /blade/ was even their on the first place.